avocado/core/settings.py

```diff
@@ -144,9 +144,15 @@ def __init__(self, config_path=None):
         :param config_path: Path to a config file. Useful for unittesting.
         """
         self.config = ConfigParser.ConfigParser()
-        self.intree = False
         self.config_paths = []
         self.config_paths_failed = []
+        _source_tree_root = os.path.dirname(os.path.dirname(os.path.dirname(
+            sys.modules[__name__].__file__)))
```

Comment:
I usually go with the simpler `__file__` instead of `sys.modules[__name__].__file__` but they're interchangeable, and the  line you're removing looks like this, so LGTM.